exam/helpers.py

```diff
@@ -160,3 +160,22 @@ def __call__(self, *args, **kwargs):
                 return return_value
 
         raise TypeError('Unknown effect for: %r, %r' % (args, kwargs))
+
+
+class sequential_effect(effect):
+    '''Subclass from exam.helpers.effect, allowing for sequential calls'''
+    
+    def __call__(self, *args, **kwargs):
+        this_call = self.call_class(*args, **kwargs)
+        for call_obj, return_value in self:
```

Comment:
Am I reading this right that *any* call in `self` that matches `*args, **kwargs` will be matched, not necessarily the call at the head of the list?